plans/dev/main.fmf

```diff
@@ -0,0 +1,28 @@
+/:
+  inherit: false
+
+execute:
```

Comment:
Let's include a short summary to make it clear what these plans are about?

Reply:
That's a branch so it wouldn't be executed, but good point, I'm making `/plans/dev/without-tmt` to make place for that.